sr_robot_launch/launch/sr_bimanual.launch

```diff
@@ -21,7 +27,7 @@
   <!-- Set to true to spawn the position controllers for the hands-->
   <arg name="hand_ctrl" default="true"/>
   <!-- The control mode PWM (true) or torque (false) -->
-  <arg name="pwm_control" default="$(optenv PWM_CONTROL 1)"/>
+  <arg name="pwm_control" default="true"/>
```

Comment:
do we need to set this to true like this?

Reply:
I did that because it was similar what we are doing in the unimanual launch file: https://github.com/shadow-robot/sr_interface/blob/melodic-devel/sr_robot_launch/launch/srhand.launch#L20